packages/adk/templates/action/typescript/README.md

```diff
@@ -17,175 +17,88 @@ The **ACTIONNAME** action synthesizes and deploys your [AWS Cloud Development Ki
 - If the action relies on other actions, include the larger workflow YAML.  
 - Example content follows. -->
 
-The following example shows the **ACTIONNAME** action deploying an AWS CDK stack called `my-app-stack` into the `us-west-2` AWS Region. The AWS CDK source files are in a repository represented by the label, `WorkflowSource`.
+The following example shows how to configure **%%ACTION_NAME%%-CI-workflow**.
 
 > **Note**:  The example is for illustrative purposes, and will not work without additional configuration.
 
 
 ```
-Name: codecatalyst-cdk-deploy-workflow
+Name: %%ACTION_NAME%%-CI-workflow
 SchemaVersion: 1.0
 ...
 Actions:      
-  ACTIONNAME:
-    Identifier: aws/cdk-deploy@v1
-    Environment:
-      Name: codecatalyst-cdk-deploy-environment
-      Connections:
-        - Name: codecatalyst-account-connection
-          Role: codecatalyst-cdk-deploy-role
+  %%ACTION_NAME%%:
+    Identifier: %%ACTION_NAME_LOWERCASE%%@v1
     Inputs:
       Sources:
         - WorkflowSource
     Configuration:
-      StackName: my-app-stack
-      Region: us-west-2
+      WhoToGreet: Name
+      HowToGreet: Hello there,
```

Comment:
We know the input configuration from the action yml, we should populate this Configuration documentation to reflect that.

Reply:
addressed